apps/sms/views/inbox/js/inbox.js

```diff
@@ -803,6 +823,20 @@ var InboxView = {
     }
   },
 
+  /**
+   * Update the DOM element for a thread.
+   *
+   * @param {Record} record contains the thread informations.
+   * @param {Object} options Various options or hints to make the update more
+   * efficient.
+   * @param {Boolean} options.conversationDraft True if this update comes from a
+   * draft change for this thread. Will be false for "isDraft" conversation
+   * nodes.
+   * @param {Boolean} options.deleted True if this update comes from a message
+   * deletion.
+   * @param {Boolean} options.unread True if we have a new unread message in
+   * this thread.
+   */
   updateThread: function inbox_updateThread(record, options) {
```

Comment:
__question(:bike:):__ wondering if we should do ```(record, options = {})``` to replace all this ```options && options.option``` check with ```options.option```?